Moderation-Policy.md

```diff
@@ -79,64 +78,73 @@ Note, however, that unfamiliarity with the [Code of Conduct][] does not excuse a
+The Node.js Moderation Team is tasked with enforcement of this policy.
 
-### Privacy of the nodejs/moderation Repository
+Moderation team members are Collaborators nominated by either the TSC or CommComm and must be approved by *both* committees. If there are objections to a specific nomination, then a simple majority vote of all TSC and CommComm members is required. Moderators are removed from the Moderation Team following the same process.
 
-The nodejs/moderation Repository is used to discuss the potentially sensitive details of any specific moderation issue. The repository is private but accessible to all Collaborators. The details of any issue discussed within the nodejs/moderation repository are expected to remain confidential and are not to be discussed in any public forum or social media service.
+Twice per month, the Moderation Team must provide a report of all Moderation actions taken by the Moderation Team to both the CommComm and TSC.
 
-Any Collaborator found to be violating the privacy of the nodejs/moderation repository by repeatedly sharing or discussing the details of nodejs/moderation issues in any public forum or social media service risks being removed from the Node.js GitHub organization through standard TSC motion and vote.
+#### Escalation of Issues
 
-### TSC Delegation of Authority to a "Moderation Working Group"
+Any Moderation issue or dispute may be escalated to the TSC and CommComm for review by tagging the original issue, pull request, or associated nodejs/moderation repository tracking issue with the `moderation-review` label. Any such Moderation action may be overturned through a simple majority vote of all TSC and CommComm members.
 
-For any Moderation issue that does not directly involve a TSC member, the TSC may choose to delegate some or all of it's Moderation-related responsibilities to a "Moderation Working Group". All members of such a Working Group must be Collaborators and the TSC will have responsibility for selecting the membership of that Moderation Working Group.
+Moderation Team members directly involved in a Moderation issue -- as either the Requester or author of the Post in question -- are expected to recuse themselves from any decisions required to resolve the issue.
 
 ### Modifications to This Policy
 
-Modifications to this policy are made through normal [TSC motion and vote](https://GitHub.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-voting). Any Collaborator may submit a PR proposing changes to this policy. Those PRs must be labeled using the `tsc-agenda` label. Including a mention to `@nodejs/tsc` can be used to call the issue to TSC's attention.
+Modifications to this policy are subject to approval by both the TSC and CommComm. If there any objections to any proposed change, a simple majority vote of all CommComm and TSC members is required.
```

Comment:
So I've been thinking about this for a while, and I'm not sure which approach I like best.

**Each Community Votes Separately**

I like this concept more than combining the members into a mega committee, but it has some issues I haven't been able to work around yet.

- I think that the voting mechanism should be left up to each committee to decide. I would recommend that we follow the normal process for changing policies, aka consensus w/ fallback to vote.  I don't think that _requiring_ a vote is necessary in most cases and will probably add unnecessary overhead.
- Each committee needs to keep track of it's own part of the discussion. This implies we would need to have an issue opened in each committee's repo to track discussion so that we don't mash votes together in the case of voting, and that we don't mash consensus between the two committees together.
    - This has the disadvantage of fracturing the conversation into two places. This may be acceptable, but I do see the possibility of different viewpoints not being considered because they originated in an issue for a committee half of the folks aren't members of.
- Some people will be involved in the discussion twice. This is less important during consensus seeking, but raises questions during voting. Should people be allowed to effectively "vote twice"?
    - On the one hand, it's possible for some individuals to have more influence than others, which can be discouraging
   - On the other hand, if someone is involved in both groups, they're probably putting more work in and have context that folks only involved in one committee don't have, so having "two votes" may actually make sense.

**Form a mega-committee from members of each committee**

This simplifies all of the problems that separating by each committee has. I think the primary downside though is that each committee brings a unique perspective to moderation, and I feel that this is something we don't want to loose in the deliberation process.